flink-filesystems/flink-hadoop-fs/src/main/java/org/apache/flink/runtime/util/HadoopUtils.java

```diff
@@ -143,4 +142,24 @@ public static boolean isMinHadoopVersion(int major, int minor) throws FlinkRunti
 
 		return maj > major || (maj == major && min >= minor);
 	}
+
+	/**
+	 * Search Hadoop configuration files in the given path, and add them to the configuration if found.
+	 */
+	private static boolean addHadoopConfIfFound(Configuration configuration, String possibleHadoopConfPath) {
+		boolean foundHadoopConfiguration = false;
```

Comment:
nit: you can inline the variable as `return true`/`return false` instead of introducing a mutable variable just for return a value.

Reply:
Good point

Reply:
I found that we can't immediately return when the first conf is found, cause the second conf should also be added to the resources.